test/stability/testDivide/testDivide.go

```diff
@@ -106,10 +109,10 @@ func main() {
 	dpdkLogLevel = *(flag.String("dpdk", "--log-level=0", "Passes an arbitrary argument to dpdk EAL"))
 	flag.Parse()
 	if err := initDPDK; err != nil {
```

Comment:
wow did you test if it should not work?! initDPDK is not called at all, always err != nil.

Reply:
Forget to git add a file - good catch, thank you!